src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -79,7 +79,6 @@ PROGRAM_INFO("Parametric Naive Bayes Classifier",
     SEE_ALSO("mlpack::naive_bayes::NaiveBayesClassifier C++ class "
         "documentation", "@doxygen/classmlpack_1_1naive__bayes_1_1"
         "NaiveBayesClassifier.html"));
-
```

Comment:
We should leave this line here too.

Reply:
Can you re-add the blank line please?  Vertical whitespace is important to logically break up the code and make it easier to read.